styles/_actions.scss

```diff
@@ -8,22 +8,24 @@
     font-size: 0;
     margin: 15px 0;
 
-    &:hover::before,
-    &:focus::before,
-    &:active::before {
+    &:hover,
+    &:focus,
+    &:active {
       background-color: $primary-shade;
     }
 
-    &::before {
-      @extend .icon;
-      background-color: transparent;
-      border-radius: 50%;
-      color: $secondary-text-color;
-      content: "\2699";
-      display: block;
-      font-size: $font-size-small;
-      margin: auto;
-      padding: 5px;
-    }
+    @extend .icon;
```

Comment:
I think we'll be killing `.icon` when all the icons are SVG. Could you replace it with the necessary styles? (probably just `vertical-align-middle`)

Reply:
Do you just mean to inline the rules from `.icon`?

Reply:
Sorry, I think I understand. I think the correct thing to do is get rid of `@extend .icon` altogether. I've anyways added:
```
  display: flex;
  align-items: center;
  justify-content: center;
```
To get centering.

Reply:
But perhaps we should make a new class with those three rules? I've added this both to `.vote-up`, `.vote-down`, and `.actions .dropdown-toggle`.

Reply:
I've removed the `@extend .icon` - also for the vote buttons.

Reply:
Thanks, that's exactly what I meant!